### ISSUE TITLE
Does not start from Home Assistant

### ISSUE THREAD

**Issue by minimag450** [hassio-addons/addon-tasmoadmin#216]
when i want to start tasmoadmin from HA it does not work. I enter my correct credentials, but after confirmation only the input fields are cleared and the startup screen appears. If I start via supervisor/ tasmoadmin/ useröberflöche it works. But it is cumbersome:-(# Problem/Motivation

HASS version 2021.11.0
tasmoadmin version 0.15.0

Danke für jede Hilfe!

Gruß, Andreas

> (Why the issue was filed)

## Expected behavior

> (What you expected to happen)

## Actual behavior

> (What actually happened)

## Steps to reproduce

> (How can someone else make/see it happen)

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)
![Screenshot_20211109-110044](https://user-images.githubusercontent.com/59397474/140904308-fac8087c-c9be-4478-905e-7100c4d43b65.jpg)
![Screenshot_20211109-110109](https://user-images.githubusercontent.com/59397474/140904314-e9f22d14-2648-4c7e-a27c-dd10c13c3fe8.jpg)
![Screenshot_20211109-110120](https://user-images.githubusercontent.com/59397474/140904319-a849d0e0-7671-4e6b-b763-2fc21bc99669.jpg)

**Comment by Slyke** [comment]
Same here. Fresh install of Home Assistant, TasmotaAdmin was my first Addon installed.

Here are the logs:
```
-----------------------------------------------------------
 Add-on: TasmoAdmin
 Centrally manage all your Sonoff-Tasmota devices
-----------------------------------------------------------
 Add-on version: 0.15.0
 You are running the latest version of this add-on.
 System: Home Assistant OS 6.6  (aarch64 / raspberrypi4-64)
 Home Assistant Core: 2021.11.2
 Home Assistant Supervisor: 2021.10.8
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] nginx.sh: executing... 
[01:52:01] FATAL: 
[01:52:01] FATAL: SSL has been enabled using the 'ssl' option,
[01:52:01] FATAL: this requires a SSL certificate file which is
[01:52:01] FATAL: configured using the 'certfile' option in the
[01:52:02] FATAL: add-on configuration.
[01:52:02] FATAL: 
[01:52:02] FATAL: Unfortunately, the file specified in the
[01:52:02] FATAL: 'certfile' option does not exists.
[01:52:02] FATAL: 
[01:52:02] FATAL: Please ensure the certificate file exists and
[01:52:02] FATAL: is placed in the '/ssl/' directory.
[01:52:02] FATAL: 
[01:52:02] FATAL: In case you don't have SSL yet, consider getting
[01:52:02] FATAL: a SSL certificate or setting the 'ssl' option
[01:52:02] FATAL: to 'false' in case you are not planning on using
[01:52:02] FATAL: SSL with this add-on.
[01:52:02] FATAL: 
[01:52:02] FATAL: Check the add-on manual for more information.
[01:52:02] FATAL: 
[cont-init.d] nginx.sh: exited 1.
[cont-finish.d] executing container finish scripts...
[cont-finish.d] 99-message.sh: executing... 
-----------------------------------------------------------
                Oops! Something went wrong.

 We are so sorry, but something went terribly wrong when
 starting or running this add-on.
 
 Be sure to check the log above, line by line, for hints.
-----------------------------------------------------------
[cont-finish.d] 99-message.sh: exited 0.
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
```

**Comment by Slyke** [comment]
> That is not the same. You have SSL incorrectly configured, as explained in the error message you pasted.

It's using HAOS. I didn't know you could configure SSL on it?

**Comment by frenck** [comment]
> It's using HAOS. I didn't know you could configure SSL on it?

It tells you... right in that error message....

**Comment by Slyke** [comment]
Ah right, didn't read it fully.

**Comment by minimag450** [comment]
I do not know what happened.  I just installed the new HASS core this morning.  Then I was working.  And right now I'm reading your answer.  Now I wanted to see what I did wrong.  I saw under Supervisor / tasmoadmin that SSL is set to false (is that correct?).  Then I tapped TasmoAdmin in the sidebar - and it works.  I didn't even have to register.  Still, thank you very much for your help.
Many thanks! Regards, Andreas
